doc/user_guide/configuration/all-options.rst

```diff
@@ -538,10 +538,12 @@ Standard Checkers
 .. code-block:: toml
 
    [tool.pylint.basic]
+   # Possible choices: ['snake_case', 'camelCase', 'PascalCase', 'UPPER_CASE', 'any']
```

Comment:
Perhaps these could be enums - making the possible values a bit easier to maintain?
for example:
```python
from enum import StrEnum


class NamingStyle(StrEnum):
    SNAKE_CASE: "snake_case"
    CAMEL_CASE: "camelCase"
    ...

attr-naming-style = NamingStyle.SNAKE_CASE.value
```